File: src/test/java/com/example/demo/StudentTest.java
package com.example.demo;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import com.example.demo.school.Student;

public class StudentTest {

    @Test
    void highestTest() {
        Student student = new Student("Annie",80, 90);
        Assertions.assertEquals(90, student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("Annie",80, 90);
        Assertions.assertEquals((80+90)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/example/kotlin/StudentTest.kt
package com.example.kotlin

import com.example.kotlin.student.Student2
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test


class StudentTest {
    @Test
    fun averageTest(){
        val stu = Student2("Annie", 80, 60)
        Assertions.assertEquals((80+60)/2, stu.getAverage())
    }

    @Test
    fun highestTest(){
        val stu = Student2("Annie", 80, 60)
        Assertions.assertEquals(80,stu.highest())
    }

    @Test
    fun gradingTest(){
        val stu = Student2("Annie", 80, 60)
        Assertions.assertEquals('C',stu.grading())
    }


}